configurations/morcore2esuites/Configuration_TaakApplicatieApi.xml

```diff
@@ -5,42 +5,65 @@
             <ApiListener
                 name="ListTaakTypes"
                 uriPattern="/taaktypes" />
-
         </Receiver>
 
         <Pipeline>
             <Exits>
                 <Exit name="Exit" state="SUCCESS" />
-            </Exits>
-
-            <EchoPipe name="Hello" getInputFromFixedValue="World!"></EchoPipe>
+                <Exit name="Reject" state="REJECTED"/>
+                <Exit name="failure" state="ERROR"/>
+             </Exits>
+            
+            <fixedResultPipe name="InputTaaktypesLijst" filename="responses/ZaakTypes.xml"/>
+
+            <XmlValidatorPipe name="ValidateInputMessage" schema="xsd/validateGetTaakTypes.xsd" root="root"/>
+                <Forward name="jsonpipe" path="XmlToJson"/>
+                <Forward name="failure" path="failure"/>
+            
+            <JsonPipe name="XmlToJson" direction="XML2JSON">
+                <Forward name="success" path="Exit"/>
+                <Forward name="failure" path="Reject"/>
```

Comment:
Reject is not fitting here. Indicating an error is clearer. This is because we do not deal with user input here, so letting the user know their input was REJECTED is incorrect.